Mathlib/Data/Array/Basic.lean

```diff
@@ -5,8 +5,11 @@ Authors: Mario Carneiro
 -/
 import Batteries.Tactic.Alias
 
+-- TODO 2024-06-07: This has been upstreamed in https://github.com/leanprover/lean4/pull/4389
```

Comment:
That PR has landed - can you reword this comment accordingly? Right now, it's not clear to me what this means.

Reply:
IMHO, it's easy enough to follow the link.
bors merge